packages/fast-tooling-react/src/message-system/message-system.props.ts

```diff
@@ -1,23 +1,43 @@
 export enum MessageSystemComponentTypeAction {
     register = "register",
     deregister = "deregister",
+    registered = "registered",
```

Comment:
I'm not quite following the naming here - "registered", "deregistered", and "duplicated" seem more like states or event types than actions. Maybe there is a way to organize these differently? You call - just caught me as inconsistent.

Reply:
I'll take a look on my next pass to see if I can clarify, but the outgoing messages are all notifications that the action has been completed with any extra parameters that should be passed. So you might have an incoming message to "register" the component, and an outgoing message confirming that the component has been "registered".